src/unix/linux_like/android/b64/mod.rs

```diff
@@ -105,6 +105,10 @@ s! {
         pub f_namemax: ::c_ulong,
         __f_spare: [::c_int; 6],
     }
+
+    pub struct sigset64_t {
+        __bits: [::c_ulong; 1]
+    }
```

Comment:
It makes more sense to move this to `s_no_extra_traits!` and implement the traits in a meaningful way (if there is one), or not implement them at all.

Reply:
Oh, okay. I'm not actually sure how those instances should be implemented. So I'll move to `s_no_extra_traits!`.

Reply:
Yes, I don't see a way to meaningful implement much for this type either, so don't worry about that.